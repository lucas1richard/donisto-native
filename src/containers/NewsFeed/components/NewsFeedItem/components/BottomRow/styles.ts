import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  bottomWrapper: {
    height: 49,
    alignItems: 'center',
    flexDirection: 'row',
    paddingHorizontal: 22,
    justifyContent: 'space-between'
  }
});

export default styles;
