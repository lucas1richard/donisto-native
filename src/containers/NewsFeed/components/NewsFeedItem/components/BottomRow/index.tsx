import React from 'react';
import { View } from 'react-native';
import { Entypo } from '@expo/vector-icons';
import Txt from 'components/Txt';
import styles from './styles';

class BottomRow extends React.Component {
  render() {
    return (
      <View style={styles.bottomWrapper}>
        <View style={{ flexDirection: 'row', alignItems: 'center' }}>
          <Entypo
            name="eye"
            color="rgb(200,199,204)"
            size={18}
          />
          <Txt style={{ fontSize: 13, marginLeft: 8 }}>4567</Txt>
        </View>
        <View style={{ flexDirection: 'row', alignItems: 'center' }}>
          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
            <Entypo
              name="heart"
              color="rgb(255,45,85)"
              size={22}
            />
            <Txt style={{ fontSize: 13, marginLeft: 8 }}>24</Txt>
          </View>
          <View style={{ flexDirection: 'row', alignItems: 'center', marginLeft: 8 }}>
            <Entypo
              name="message"
              color="rgb(200,199,204)"
              size={22}
            />
            <Txt style={{ fontSize: 13, marginLeft: 8 }}>2</Txt>
          </View>
        </View>
      </View>
    );
  }
}

export default BottomRow;