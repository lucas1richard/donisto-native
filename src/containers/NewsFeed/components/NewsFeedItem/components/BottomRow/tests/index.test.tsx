import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import BottomRow from '..';

describe('<BottomRow />', () => {
  const renderComponent = () => toJson(shallow(<BottomRow />));
  it('renders', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});