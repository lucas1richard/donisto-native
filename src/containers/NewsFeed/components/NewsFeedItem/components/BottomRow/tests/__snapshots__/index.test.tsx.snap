// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BottomRow /> renders 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "height": 49,
      "justifyContent": "space-between",
      "paddingHorizontal": 22,
    }
  }
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="rgb(200,199,204)"
      name="eye"
      size={18}
    />
    <Txt
      color="default"
      style={
        Object {
          "fontSize": 13,
          "marginLeft": 8,
        }
      }
    >
      4567
    </Txt>
  </Component>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <Icon
        allowFontScaling={false}
        color="rgb(255,45,85)"
        name="heart"
        size={22}
      />
      <Txt
        color="default"
        style={
          Object {
            "fontSize": 13,
            "marginLeft": 8,
          }
        }
      >
        24
      </Txt>
    </Component>
    <Component
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "marginLeft": 8,
        }
      }
    >
      <Icon
        allowFontScaling={false}
        color="rgb(200,199,204)"
        name="message"
        size={22}
      />
      <Txt
        color="default"
        style={
          Object {
            "fontSize": 13,
            "marginLeft": 8,
          }
        }
      >
        2
      </Txt>
    </Component>
  </Component>
</Component>
`;
